pom.xml

```diff
@@ -238,12 +238,9 @@
         <!-- Derby driver used by JBatch -->
         <version.org.apache.derby>10.13.1.1</version.org.apache.derby>
 
-        <!--
-            Using JAXB 2.3.0 as 2.2.11 seems to be broken in more than a few ways.
-            In particular version 2.2.11 of the impl depends on version 2.2.12-b140109.1041 of the API.
-         -->
-        <version.javax.xml.bind.jaxb-api>2.3.0</version.javax.xml.bind.jaxb-api>
-        <version.org.glassfish.jaxb>2.3.0.1</version.org.glassfish.jaxb>
+        <!-- This should be aligned with the version used in ORM -->
+        <!-- HSEARCH-3237: this should be removed when we remove the dependency from the Elasticsearch module to JAXB -->
+        <version.javax.xml.bind.jaxb-api>2.2.12-b140109.1041</version.javax.xml.bind.jaxb-api>
```

Comment:
Didn't you upgrade to 2.3 in ORM? Or you plan to upgrade it when moving to CR2?

Reply:
I'll try to remember to upgrade when moving to CR2, but I remove this in one of the following commits anyway.
To be clear, as mentioned above, this PR is on hold precisely because I know CR1 has bugs that make our build (JDK10, and also JDK11) fail. So I won't merge it until CR2 is out and I updated the PR to target CR2 instead of CR1.